Library/ViewModels/FindFriendsFriendFollowCellViewModel.swift

```diff
@@ -78,10 +78,10 @@ public final class FindFriendsFriendFollowCellViewModel: FindFriendsFriendFollow
       .map { $0.stats.backedProjectsCount ?? 0 }
       .map(Strings.social_following_friend_projects_count_backed(backed_count:))
 
-    self.hideProjectsCreated = friend.map { $0.stats.createdProjectsCount == 0 }
```

Comment:
We might show the storyboard "1 created" string if the `createdProjectsCount` is `nil`! This is because we're only doing a strict equality check against `0` and not accounting for `nil`.